classes/Utils.php

```diff
@@ -1120,6 +1120,25 @@ public function check_nonce() {
 		}
 	}
 
+	/**
+	 * Check if the openai response has any error or not.
+	 * If there any error then send the error response, otherwise continue.
+	 *
+	 * @param array $response The openai response.
+	 * @return object
```

Comment:
I think the return type would be mixed: object|null. `send_response` will just echo json data and die, in that case null will be returned.